onto_spread_ed/release/MergeReleaseStep.py

```
@@ -10,19 +16,52 @@
         builder = RobotOntologyBuildService()
         result = Result(())
 
-        files = [(n, f) for n, f in self._release_script.files.items() if all(s.type == "owl" for s in f.sources)]
+        merge_files = [(n, f) for n, f in self._release_script.files.items() if
+                       all(s.type == "owl" for s in f.sources) and (n in self._files or f.target.publish)]
+        collapse_imports = [(n, f) for n, f in self._release_script.files.items() if
+                            (n in self._files or f.target.publish) and (n, f) not in merge_files]
+
+        self._total_items = len(merge_files) + len(collapse_imports)
+
+        for k, file in merge_files:
```

Ruff

Loop control variable `k` not used within loop body

Rename unused `k` to `_k`

(B007)